FormBuilder.cs

```diff
@@ -218,6 +156,50 @@ public void Save()
 			}
 		}
 
+		void LoadTerrainDefinitions()
+		{
+			var terrainDefinition = new Dictionary<string, TerrainTypeInfo>();
+			var yaml = MiniYaml.DictFromFile("defaults.yaml");
+			terrainDefinition = yaml["Terrain"].ToDictionary().Values.Select(y => new TerrainTypeInfo(y)).ToDictionary(t => t.Type);
+			var i = 0;
```

Comment:
I've came up with this method name based on the first variable. Can you suggest something else  ?